Changelog.md

```diff
@@ -1,136 +1,148 @@
-# 2.7.1
+# Spectrometer Changelog
+
+## Unreleased
+
+- Add support and documentation for user-defined dependencies ([#245](https://github.com/fossas/spectrometer/pull/245))
+- Allow using `.yml` or `.yaml` extensions for `fossa-deps` file, but not both ([#245](https://github.com/fossas/spectrometer/pull/245))
+- `fossa-deps` file is checked before running discovery/analysis, and is no longer run in parallel with other analysis functions ([#245](https://github.com/fossas/spectrometer/pull/245))
```

Comment:
[nit] for changelog entries, unlike commit messages, use e.g. `Adds support ...` and `Allows ...`